sway-ast/src/keywords.rs

```diff
@@ -142,93 +146,121 @@ macro_rules! define_token (
     };
 );
 
-define_token!(SemicolonToken, "a semicolon", [Semicolon], []);
+define_token!(SemicolonToken, "a semicolon", ";", [Semicolon], []);
 define_token!(
     ForwardSlashToken,
     "a forward slash",
+    "/",
     [ForwardSlash],
     [Equals]
 );
 define_token!(
     DoubleColonToken,
     "a double colon (::)",
+    "::",
     [Colon, Colon],
     [Colon]
 );
-define_token!(StarToken, "an asterisk (*)", [Star], [Equals]);
-define_token!(DoubleStarToken, "`**`", [Star, Star], []);
-define_token!(CommaToken, "a comma", [Comma], []);
-define_token!(ColonToken, "a colon", [Colon], [Colon]);
+define_token!(StarToken, "an asterisk (*)", "*", [Star], [Equals]);
+define_token!(DoubleStarToken, "`**`", "**", [Star, Star], []);
+define_token!(CommaToken, "a comma", ",", [Comma], []);
+define_token!(ColonToken, "a colon", ":", [Colon], [Colon]);
 define_token!(
     RightArrowToken,
     "`->`",
+    "->",
     [Sub, GreaterThan],
     [GreaterThan, Equals]
 );
-define_token!(LessThanToken, "`<`", [LessThan], [LessThan, Equals]);
+define_token!(LessThanToken, "`<`", "<", [LessThan], [LessThan, Equals]);
 define_token!(
     GreaterThanToken,
     "`>`",
+    ">",
     [GreaterThan],
     [GreaterThan, Equals]
 );
-define_token!(OpenAngleBracketToken, "`<`", [LessThan], []);
-define_token!(CloseAngleBracketToken, "`>`", [GreaterThan], []);
-define_token!(EqToken, "`=`", [Equals], [GreaterThan, Equals]);
-define_token!(AddEqToken, "`+=`", [Add, Equals], []);
-define_token!(SubEqToken, "`-=`", [Sub, Equals], []);
-define_token!(StarEqToken, "`*=`", [Star, Equals], []);
-define_token!(DivEqToken, "`/=`", [ForwardSlash, Equals], []);
-define_token!(ShlEqToken, "`<<=`", [LessThan, LessThan, Equals], []);
-define_token!(ShrEqToken, "`>>=`", [GreaterThan, GreaterThan, Equals], []);
+define_token!(OpenAngleBracketToken, "`<`", "<", [LessThan], []);
+define_token!(CloseAngleBracketToken, "`>`", ">", [GreaterThan], []);
```

Comment:
And here:

    define_token!(CloseAngleBracketToken, "`>`", ">", [GreaterThan], [GreaterThan, Equals]);